### PR TITLE
[WIP] Adding Docstrings to the code.

### PR DESCRIPTION
**Pull request by ashutosh-b-b**

### CODE REVIEW

File: src/NeuralNetDiffEq.jl [hunk 1]
```diff
@@ -36,6 +47,18 @@ function Base.show(io::IO, A::TerminalPDEProblem)
...
+A standard Kolmogorov PDE Problem.
+# Arguments
+* `f` : The drift function from Feynman-Kac representation of the PDE.
+* `g` : The noise function from Feynman-Kac representation of the PDE.
+* `phi` : The terminal condition for the PDE.
+* `tspan`: The timespan for the problem.
+* `xspan`: The xspan for the problem.
+* `d`: The dimensions of the input x.
+* `noise_rate_prototype`: A prototype type instance for the noise rates, that is the output g.
+"""
```
Comment (ChrisRackauckas) [inline src/NeuralNetDiffEq.jl:62]:
```suggestion
```

File: src/NeuralNetDiffEq.jl [hunk 2]
```diff
@@ -36,6 +47,18 @@ function Base.show(io::IO, A::TerminalPDEProblem)
   print(io,"timespan: ")
   show(io,A.tspan)
 end
+"""
```
Comment (ChrisRackauckas) [inline src/NeuralNetDiffEq.jl:50]:
```suggestion
end

"""
```

**Comment by ashutosh-b-b** [comment]
@Chris should I add about the upper and lower limit as well in the `TerminalPDEProblem` or should I add it to the NNPDEHan docstring?

**Comment by ChrisRackauckas** [comment]
It should be documented in `TerminalPDEProblem` and we should make sure to update the other solver to allow for bounds as well.

**Comment by ashutosh-b-b** [comment]
> It should be documented in `TerminalPDEProblem` and we should make sure to update the other solver to allow for bounds as well.

Which other solver?

**Comment by ChrisRackauckas** [comment]
NNPDENS

**Comment by ChrisRackauckas** [comment]
Let's merge and keep moving. @zzj0402 should make note of this.

**Comment by ChrisRackauckas** [comment]
Another PR should add the other missing docstrings and explain the lower/upper bounds parts.

**Pull request merged**